content/webapp/components/Exhibition/Exhibition.tsx

```diff
@@ -207,38 +175,23 @@ function getAccessibilityItems(
       icon: accessibility,
     },
   ];
-  if (exhibitionAccessContent) {
-    return accessibilityItems.filter(item => {
-      if (item.description[0] && 'text' in item.description[0]) {
-        return item.description[0]?.text !== a11y.largePrintGuides;
-      } else {
-        return true;
-      }
-    });
-  } else {
-    return accessibilityItems.filter(item => {
-      if (item.description[0] && 'text' in item.description[0]) {
-        return (
-          item.description[0]?.text !== a11y.bsl &&
-          item.description[0]?.text !== a11y.accessResources
-        );
-      } else {
-        return true;
-      }
-    });
-  }
+
+  return accessibilityItems.filter(item => {
```

Comment:
we can remove the a11y.largePrintGuides for accessibilityItems and so don't need to filter them out. In fact, once that is done, we can remove the getAccessibilityItems function and just use accessibilityItems directly